src/main/kotlin/solve/scene/view/virtualizedfx/VirtualizedFXGrid.kt

```diff
@@ -48,6 +52,29 @@ class VirtualizedFXGrid(
         }
     }
 
+    fun changeColumnsNumber(newColumnsNumber: Int) {
```

Comment:
We can work with grid only through the interface

Reply:
Why not just redraw scene?

Reply:
> Why not just redraw scene?
We lose the highlited landmarks so user and therefore, the user most likely will not change the grid size during he is working